ui/src/app/store/machine/slice.ts

```diff
@@ -505,21 +507,6 @@ const machineSlice = createSlice({
     cloneSuccess: statusHandlers.clone.success,
     [NodeActions.COMMISSION]: {
       prepare: (params: CommissionParams) => {
-        let formattedCommissioningScripts: (string | Script["id"])[] = [];
-        if (
-          params.commissioningScripts &&
-          params.commissioningScripts.length > 0
-        ) {
-          formattedCommissioningScripts = params.commissioningScripts.map(
-            (script) => script.id
-          );
-          if (params.updateFirmware) {
-            formattedCommissioningScripts.push("update_firmware");
-          }
-          if (params.configureHBA) {
-            formattedCommissioningScripts.push("configure_hba");
-          }
-        }
```

Comment:
I decided to move all this logic from the reducer to the commission form, since it's really only useful for that particular form. This way the action creator is more closely aligned to the API and could more easily be used in different contexts.